cdhweb/static_src/global/css-variables/palette.scss

```diff
@@ -2,14 +2,37 @@
   --color-black: black;
   --color-white: white;
 
-  // UPDATE_ME
-  // Add your brand-specific colours here
-  // --color-primary: whatever;
-  // etc
-
-  // UPDATE_ME probably.
-  --color-text: var(--color-black);
-  --color-link-text: #337ab7;
-  --color-link-underline: currentColor;
-  --color-link-text-hover: #115ea2;
+  // TODO remove/comment-out any unused at the end of build
+
+  // Brand
+  --color-brand-5: #f5feff;
+  --color-brand-10: #dffcff;
+  --color-brand-20: #caf9fd;
+  --color-brand-40: #a8f9ff;
+  --color-brand-60: #83f6ff;
+  --color-brand-80: #51f3ff;
+  --color-brand-100: #00edff;
+  --color-brand-110: #04cad9;
+  --color-brand-120: #1fa4ae;
+
+  // Secondary
+  --color-forest-80: #9dc359;
+  --color-forest-100: #85b42f;
+  --color-orange-60: #fea571;
+  --color-orange-100: #e34f26;
+  --color-red-100: #ed0000;
+
+  --color-grey-5: #f2f2f2;
```

Comment:
Is `grey-0` left out on purpose?

Reply:
yes, ditto 120. these are 'black' and 'white', which is just simpler for my brain to deal with :)